python/ray/autoscaler/v2/instance_manager/config.py

```diff
@@ -128,6 +128,8 @@ class NodeTypeConfig:
     min_worker_nodes: int
     # The maximal number of worker nodes can be launched for this node type.
     max_worker_nodes: int
+    # Idle timeout seconds for worker nodes of this node type.
+    idle_timeout_s: Optional[float] = None
```

Comment:
nit: should we enforce it as integer with a cast when we add this? I see it being int as part of the schema

Or we could make this a float in the schema too. No preference over this.

Reply:
I'll change it to a `number` type in the schema and then add a cast to float when we call `idle_timeout_s = group_spec.get(IDLE_SECONDS_KEY)`, since I implemented it as an int in the RayCluster CRD for consistency with the other field: https://github.com/ray-project/kuberay/blob/925effe34022c72c41691c0b79d8d3051d4a1b77/ray-operator/apis/ray/v1/raycluster_types.go#L94

Reply:
I ran the tests again and implemented this change in: 1bd8afb

Reply:
Awesome, thanks for the great work!